zazu/plugins/eslint_styler.py

```diff
@@ -0,0 +1,66 @@
+# -*- coding: utf-8 -*-
+"""ESLint plugin for zazu."""
+import zazu.styler
+zazu.util.lazy_import(locals(), [
+    'json',
+    'os',
+    'subprocess'
+])
+
+__author__ = "Patrick Moore"
+__copyright__ = "Copyright 2018"
+
+
+class ESLintStyler(zazu.styler.Styler):
+    """ESLint plugin for code styling."""
+
+    def style_string(self, string, filepath):
+        """Fix a string to be within style guidelines.
+
+        Args:
+            string (str): the string to style
+            filepath (str): the filepath of the file being styled
+
+        Returns:
+            Styled string.
+
+        """
+
+        node_modules_path = None
+        eslint = 'eslint'
+        cwd = os.path.normpath(os.getcwd())
+        dirname = os.path.normpath(os.path.dirname(filepath))
+        loop_count = 0
+
+        while True and loop_count < 100:
```

Comment:
I think that the loop_count check failure should probably be fatal...I can't think of a circumstance under which we should hit it if we've done everything else right. I'd raise a click.ClickException in that case.

Also some of these paths are not covered. Probably want a test that uses a local eslint.js file, one that hits the file system boundary and one that hits the cwd boundary and one that hits the loop_count boundary.